### DISCUSSION TITLE
CSS for hiding toolbar not working on macOS

### DISCUSSION THREAD

**Discussion by 0hwow** [Zettlr/Zettlr#2969] [Q&A] [answered]
Hi,
The custom CSS code provided [here](https://docs.zettlr.com/en/core/custom-css) to hide the toolbar is not working:
`body.show-menubar #editor.fullscreen, #editor.fullscreen, .CodeMirror-fullscreen {
  top: 0px;
}`
Thanks

**Comment by nathanlesage** [comment, marked as answer]
In Zettlr 2.0 the toolbar cannot be hidden purely with CSS anymore since all of that is within a splitview component. But it could be made possible again with a corresponding setting. Feel free to open an issue.

**Reply by nathanlesage** [reply]
I've just added a setting to programmatically hide the toolbar in distraction free mode.

**Reply by nathanlesage** [reply]
Wait for the next release to come around. It will include this new setting.
